go/worker/computeenable/init.go

```diff
@@ -0,0 +1,25 @@
+package computeenable
```

Comment:
Ugh, this seems ugly. In light of my previous comment around naming we could also do something like this:
* Rename existing `compute` to `executor` (or some similar name as mentioned above).
* Restructure workers to have `go/worker/compute/{txnscheduler,executor,merge}`.
* Have this "computeenable" thing in `go/worker/compute` potentially with some common code that initializes all three workers.

Reply:
#2525 for the first 2 bullet points

Reply:
moved computeenable to compute. initialization stuff is mostly in executor, with that being the one that needs a registration callback (to wait for TEE capability)